syncthing panic v0.11.8 windows x64

```
[PMPXX] 10:57:21 INFO: syncthing v0.11.8 (go1.4.2 windows-amd64 default) unknown-user@syncthing-builder 2015-06-07 10:57:26 UTC
[PMPXX] 10:57:21 INFO: My ID: <DELETED>
[PMPXX] 10:57:21 INFO: Database block cache capacity 65536 KiB
[PMPXX] 10:57:21 INFO: Starting web GUI on http://127.0.0.1:8080/
...
Panic at 2015-06-08T10:57:23+04:00
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x58 pc=0x5b62ee]

goroutine 51 [running]:
github.com/syncthing/syncthing/internal/model.func·008()
	/go/src/github.com/syncthing/syncthing/internal/model/model.go:1192 +0x23e
created by github.com/syncthing/syncthing/internal/model.(*Model).ScanFolders
	/go/src/github.com/syncthing/syncthing/internal/model/model.go:1195 +0x6b0

goroutine 1 [runnable]:
github.com/syndtr/goleveldb/leveldb/table.func·003(0xc08a2ff600, 0xc0820b76c0, 0xc0864a4280)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/table/reader.go:621
github.com/syndtr/goleveldb/leveldb/cache.(*Cache).Get(0xc0820b76c0, 0x6ae, 0x2009f3, 0xc0832b76f8, 0x0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/cache/cache.go:385 +0x1f4
github.com/syndtr/goleveldb/leveldb/cache.(*CacheGetter).Get(0xc086ff2090, 0x2009f3, 0xc0832b76f8, 0xc0820f6150)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/cache/cache.go:56 +0x4f
github.com/syndtr/goleveldb/leveldb/table.(*Reader).readBlockCached(0xc086fc00d0, 0x2009f3, 0x4bd6, 0xc082020101, 0x823e83, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/table/reader.go:628 +0x105
github.com/syndtr/goleveldb/leveldb/table.(*Reader).getIndexBlock(0xc086fc00d0, 0x82eb01, 0xc0820b7740, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/table/reader.go:708 +0x88
github.com/syndtr/goleveldb/leveldb/table.(*Reader).NewIterator(0xc086fc00d0, 0xc08a81a0c0, 0x0, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/table/reader.go:799 +0x1ef
github.com/syndtr/goleveldb/leveldb.(*tOps).newIterator(0xc0820b9280, 0xc0820fe9b0, 0xc08a81a0c0, 0x0, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/table.go:408 +0x1da
github.com/syndtr/goleveldb/leveldb.(*tFilesArrayIndexer).Get(0xc089e46940, 0x0, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/table.go:263 +0x12d
github.com/syndtr/goleveldb/leveldb/iterator.(*arrayIteratorIndexer).Get(0xc086d8c000, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/iterator/array_iter.go:164 +0x82
github.com/syndtr/goleveldb/leveldb/iterator.(*indexedIterator).setData(0xc086d96000)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/iterator/indexed_iter.go:39 +0x70
github.com/syndtr/goleveldb/leveldb/iterator.(*indexedIterator).First(0xc086d96000, 0x137b78)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/iterator/indexed_iter.go:88 +0xd9
github.com/syndtr/goleveldb/leveldb/iterator.(*mergedIterator).First(0xc086220080, 0xc0ffffffff)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/iterator/merged_iter.go:72 +0x115
github.com/syndtr/goleveldb/leveldb/iterator.(*mergedIterator).Next(0xc086220080, 0xc0832b7c30)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/iterator/merged_iter.go:157 +0xad
github.com/syndtr/goleveldb/leveldb.(*dbIter).Next(0xc083e98000, 0xc0832b7f30)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db_iter.go:214 +0xa5
github.com/syncthing/syncthing/internal/db.ldbGenericReplace(0xc08200a2c0, 0xc08a7dcba0, 0x7, 0x8, 0xc08a8263e0, 0x20, 0x20, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/syncthing/syncthing/internal/db/leveldb.go:201 +0x5eb
github.com/syncthing/syncthing/internal/db.ldbReplace(0xc08200a2c0, 0xc08a7dcba0, 0x7, 0x8, 0xc08a8263e0, 0x20, 0x20, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/syncthing/syncthing/internal/db/leveldb.go:321 +0xaa
github.com/syncthing/syncthing/internal/db.(*FileSet).Replace(0xc083eb4040, 0x292c4bc5296d2a53, 0xfa4f3116c6a74ffc, 0x6399106f25d30469, 0x9c430056d8f4b42f, 0x0, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/internal/db/set.go:80 +0x3ae
github.com/syncthing/syncthing/internal/model.(*Model).Index(0xc082f39680, 0x292c4bc5296d2a53, 0xfa4f3116c6a74ffc, 0x6399106f25d30469, 0x9c430056d8f4b42f, 0xc0820e2748, 0x7, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/syncthing/syncthing/internal/model/model.go:507 +0xc2d
main.syncthingMain()
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/main.go:612 +0x3739
main.main()
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/main.go:378 +0x2325

goroutine 5 [syscall]:
os/signal.loop()
	c:/go/src/os/signal/signal_unix.go:21 +0x26
created by os/signal.init·1
	c:/go/src/os/signal/signal_unix.go:27 +0x3c

goroutine 7 [chan receive]:
main.trackCPUUsage()
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui_windows.go:37 +0x496
created by main.init·2
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui_windows.go:17 +0x2c

goroutine 8 [select]:
github.com/thejerf/suture.(*Supervisor).Serve(0xc0820da000)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:411 +0xf6b
created by github.com/thejerf/suture.(*Supervisor).ServeBackground
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:373 +0x39

goroutine 10 [chan receive]:
github.com/syncthing/syncthing/internal/config.(*Wrapper).Serve(0xc082086200)
	/go/src/github.com/syncthing/syncthing/internal/config/wrapper.go:84 +0x7c
created by github.com/syncthing/syncthing/internal/config.Wrap
	/go/src/github.com/syncthing/syncthing/internal/config/wrapper.go:59 +0x294

goroutine 11 [select]:
github.com/syndtr/goleveldb/leveldb/util.(*BufferPool).drain(0xc0820602a0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/util/buffer_pool.go:205 +0x225
created by github.com/syndtr/goleveldb/leveldb/util.NewBufferPool
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/util/buffer_pool.go:236 +0x253

goroutine 12 [select]:
github.com/syndtr/goleveldb/leveldb.(*DB).compactionError(0xc08200a2c0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db_compaction.go:120 +0x493
created by github.com/syndtr/goleveldb/leveldb.openDB
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db.go:126 +0x850

goroutine 13 [select]:
github.com/syndtr/goleveldb/leveldb.(*DB).mpoolDrain(0xc08200a2c0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db_state.go:73 +0x151
created by github.com/syndtr/goleveldb/leveldb.openDB
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db.go:127 +0x86a

goroutine 14 [select]:
github.com/syndtr/goleveldb/leveldb.(*DB).tCompaction(0xc08200a2c0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db_compaction.go:812 +0x85f
created by github.com/syndtr/goleveldb/leveldb.openDB
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db.go:130 +0x8ae

goroutine 15 [select]:
github.com/syndtr/goleveldb/leveldb.(*DB).mCompaction(0xc08200a2c0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db_compaction.go:759 +0x28a
created by github.com/syndtr/goleveldb/leveldb.openDB
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db.go:131 +0x8c8

goroutine 16 [select]:
github.com/syndtr/goleveldb/leveldb.(*DB).jWriter(0xc08200a2c0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db_write.go:37 +0x19e
created by github.com/syndtr/goleveldb/leveldb.openDB
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db.go:132 +0x8e2

goroutine 33 [semacquire]:
sync.(*WaitGroup).Wait(0xc0833ca000)
	c:/go/src/sync/waitgroup.go:132 +0x170
github.com/syncthing/syncthing/internal/model.(*Model).ScanFolders(0xc082f39680, 0xc083f16180)
	/go/src/github.com/syncthing/syncthing/internal/model/model.go:1197 +0x6fa
main.(*apiSvc).postDBScan(0xc083048480, 0x110aa40, 0xc083e88000, 0xc08383a000)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:785 +0x387
main.*apiSvc.(main.postDBScan)·fm(0x110aa40, 0xc083e88000, 0xc08383a000)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:158 +0x4c
net/http.HandlerFunc.ServeHTTP(0xc0823ba990, 0x110aa40, 0xc083e88000, 0xc08383a000)
	c:/go/src/net/http/server.go:1265 +0x48
net/http.(*ServeMux).ServeHTTP(0xc083ea2a20, 0x110aa40, 0xc083e88000, 0xc08383a000)
	c:/go/src/net/http/server.go:1541 +0x184
main.func·002(0x110aa40, 0xc083e88000, 0xc08383a000)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:230 +0x14f
net/http.HandlerFunc.ServeHTTP(0xc083ea4a00, 0x110aa40, 0xc083e88000, 0xc08383a000)
	c:/go/src/net/http/server.go:1265 +0x48
main.func·005(0x110aa40, 0xc083e88000, 0xc08383a000)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:283 +0x1e4
net/http.HandlerFunc.ServeHTTP(0xc0823baa70, 0x110aa40, 0xc083e88000, 0xc08383a000)
	c:/go/src/net/http/server.go:1265 +0x48
net/http.(*ServeMux).ServeHTTP(0xc083ea2a80, 0x110aa40, 0xc083e88000, 0xc08383a000)
	c:/go/src/net/http/server.go:1541 +0x184
main.func·009(0x110aa40, 0xc083e88000, 0xc08383a000)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui_csrf.go:63 +0x4ef
net/http.HandlerFunc.ServeHTTP(0xc083ea4c00, 0x110aa40, 0xc083e88000, 0xc08383a000)
	c:/go/src/net/http/server.go:1265 +0x48
main.func·006(0x110aa40, 0xc083e88000, 0xc08383a000)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:290 +0xb5
net/http.HandlerFunc.ServeHTTP(0xc0823baae0, 0x110aa40, 0xc083e88000, 0xc08383a000)
	c:/go/src/net/http/server.go:1265 +0x48
net/http.serverHandler.ServeHTTP(0xc0820f9380, 0x110aa40, 0xc083e88000, 0xc08383a000)
	c:/go/src/net/http/server.go:1703 +0x1a1
net/http.(*conn).serve(0xc083e86000)
	c:/go/src/net/http/server.go:1204 +0xb5e
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 66 [semacquire]:
sync.(*Cond).Wait(0xc083ea0980)
	c:/go/src/sync/cond.go:62 +0xa5
github.com/syncthing/syncthing/internal/events.(*BufferedSubscription).Since(0xc083ea83c0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/internal/events/events.go:245 +0x111
main.(*apiSvc).getEvents(0xc083048480, 0x110aa40, 0xc0859000a0, 0xc0861541a0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:691 +0x23d
main.*apiSvc.(main.getEvents)·fm(0x110aa40, 0xc0859000a0, 0xc0861541a0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:136 +0x4c
net/http.HandlerFunc.ServeHTTP(0xc0823ba860, 0x110aa40, 0xc0859000a0, 0xc0861541a0)
	c:/go/src/net/http/server.go:1265 +0x48
net/http.(*ServeMux).ServeHTTP(0xc083ea29c0, 0x110aa40, 0xc0859000a0, 0xc0861541a0)
	c:/go/src/net/http/server.go:1541 +0x184
main.func·002(0x110aa40, 0xc0859000a0, 0xc0861541a0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:228 +0xd3
net/http.HandlerFunc.ServeHTTP(0xc083ea4a00, 0x110aa40, 0xc0859000a0, 0xc0861541a0)
	c:/go/src/net/http/server.go:1265 +0x48
main.func·005(0x110aa40, 0xc0859000a0, 0xc0861541a0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:283 +0x1e4
net/http.HandlerFunc.ServeHTTP(0xc0823baa70, 0x110aa40, 0xc0859000a0, 0xc0861541a0)
	c:/go/src/net/http/server.go:1265 +0x48
net/http.(*ServeMux).ServeHTTP(0xc083ea2a80, 0x110aa40, 0xc0859000a0, 0xc0861541a0)
	c:/go/src/net/http/server.go:1541 +0x184
main.func·009(0x110aa40, 0xc0859000a0, 0xc0861541a0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui_csrf.go:52 +0x419
net/http.HandlerFunc.ServeHTTP(0xc083ea4c00, 0x110aa40, 0xc0859000a0, 0xc0861541a0)
	c:/go/src/net/http/server.go:1265 +0x48
main.func·006(0x110aa40, 0xc0859000a0, 0xc0861541a0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:290 +0xb5
net/http.HandlerFunc.ServeHTTP(0xc0823baae0, 0x110aa40, 0xc0859000a0, 0xc0861541a0)
	c:/go/src/net/http/server.go:1265 +0x48
net/http.serverHandler.ServeHTTP(0xc0820f9380, 0x110aa40, 0xc0859000a0, 0xc0861541a0)
	c:/go/src/net/http/server.go:1703 +0x1a1
net/http.(*conn).serve(0xc08440e280)
	c:/go/src/net/http/server.go:1204 +0xb5e
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 65 [IO wait]:
net.(*pollDesc).Wait(0xc085fba170, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08229c008, 0xc085fba060, 0xa5a7b0, 0x7, 0xba7b90, 0xc0843050fe, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc085fba000, 0xc08350b000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc0836e4010, 0xc08350b000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
bufio.(*Reader).Read(0xc0820f80c0, 0xc08350b000, 0x1000, 0x1000, 0xffffffff, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:164 +0x141
main.(*WrappedConnection).Read(0xc0853201c0, 0xc08350b000, 0x1000, 0x1000, 0x44f059, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/tls.go:123 +0x7a
net/http.(*liveSwitchReader).Read(0xc08440e228, 0xc08350b000, 0x1000, 0x1000, 0xc08207e700, 0x0, 0x0)
	c:/go/src/net/http/server.go:214 +0xb2
io.(*LimitedReader).Read(0xc0853201e0, 0xc08350b000, 0x1000, 0x1000, 0xc0849cb8e8, 0x0, 0x0)
	c:/go/src/io/io.go:408 +0xd5
bufio.(*Reader).fill(0xc083936120)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).ReadSlice(0xc083936120, 0xa, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:295 +0x25e
bufio.(*Reader).ReadLine(0xc083936120, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:324 +0x69
net/textproto.(*Reader).readLineSlice(0xc086204030, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:55 +0xa5
net/textproto.(*Reader).ReadLine(0xc086204030, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:36 +0x56
net/http.ReadRequest(0xc083936120, 0xc086154750, 0x0, 0x0)
	c:/go/src/net/http/request.go:598 +0xd2
net/http.(*conn).readRequest(0xc08440e1e0, 0x0, 0x0, 0x0)
	c:/go/src/net/http/server.go:586 +0x276
net/http.(*conn).serve(0xc08440e1e0)
	c:/go/src/net/http/server.go:1162 +0x6a5
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 20 [select]:
github.com/syncthing/syncthing/internal/model.(*ProgressEmitter).Serve(0xc0830df500)
	/go/src/github.com/syncthing/syncthing/internal/model/progressemitter.go:49 +0x8f9
created by github.com/syncthing/syncthing/internal/model.NewModel
	/go/src/github.com/syncthing/syncthing/internal/model/model.go:132 +0xb01

goroutine 21 [IO wait]:
net.(*pollDesc).Wait(0xc0847d05f0, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08229c008, 0xc0847d04e0, 0xa5a7b0, 0x7, 0xba7b90, 0x1, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc0847d0480, 0xc08416b000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc086248098, 0xc08416b000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
bufio.(*Reader).fill(0xc083936420)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).Peek(0xc083936420, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:132 +0xf7
main.(*DowngradingListener).Accept(0xc083ea4960, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/tls.go:105 +0x13e
net/http.(*Server).Serve(0xc0820f9380, 0x1109198, 0xc083ea4960, 0x0, 0x0)
	c:/go/src/net/http/server.go:1728 +0x99
main.(*apiSvc).Serve(0xc083048480)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:215 +0x1911
github.com/thejerf/suture.func·007()
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:516 +0xfd
created by github.com/thejerf/suture.(*Supervisor).runService
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:519 +0x125

goroutine 22 [select]:
github.com/syncthing/syncthing/internal/events.(*Subscription).Poll(0xc083ea49e0, 0xdf8475800, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/syncthing/syncthing/internal/events/events.go:183 +0x370
github.com/syncthing/syncthing/internal/events.(*BufferedSubscription).pollingLoop(0xc083ea83c0)
	/go/src/github.com/syncthing/syncthing/internal/events/events.go:220 +0x4f
created by github.com/syncthing/syncthing/internal/events.NewBufferedSubscription
	/go/src/github.com/syncthing/syncthing/internal/events/events.go:214 +0x299

goroutine 23 [select]:
github.com/thejerf/suture.(*Supervisor).Serve(0xc0820b0cc0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:411 +0xf6b
created by github.com/thejerf/suture.(*Supervisor).ServeBackground
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:373 +0x39

goroutine 24 [select]:
main.(*folderSummarySvc).listenForUpdates(0xc0820b5140)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/summarysvc.go:67 +0x5cd
main.*folderSummarySvc.(main.listenForUpdates)·fm()
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/summarysvc.go:47 +0x2e
main.serviceFunc.Serve(0xc0830857e0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/summarysvc.go:202 +0x27
github.com/thejerf/suture.func·007()
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:516 +0xfd
created by github.com/thejerf/suture.(*Supervisor).runService
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:519 +0x125

goroutine 25 [select]:
main.(*folderSummarySvc).calculateSummaries(0xc0820b5140)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/summarysvc.go:115 +0x2eb
main.*folderSummarySvc.(main.calculateSummaries)·fm()
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/summarysvc.go:48 +0x2e
main.serviceFunc.Serve(0xc0830857f0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/summarysvc.go:202 +0x27
github.com/thejerf/suture.func·007()
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:516 +0xfd
created by github.com/thejerf/suture.(*Supervisor).runService
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:519 +0x125

goroutine 49 [semacquire]:
sync.(*Cond).Wait(0xc083ea0980)
	c:/go/src/sync/cond.go:62 +0xa5
github.com/syncthing/syncthing/internal/events.(*BufferedSubscription).Since(0xc083ea83c0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/internal/events/events.go:245 +0x111
main.(*apiSvc).getEvents(0xc083048480, 0x110aa40, 0xc083e880a0, 0xc08383a0d0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:691 +0x23d
main.*apiSvc.(main.getEvents)·fm(0x110aa40, 0xc083e880a0, 0xc08383a0d0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:136 +0x4c
net/http.HandlerFunc.ServeHTTP(0xc0823ba860, 0x110aa40, 0xc083e880a0, 0xc08383a0d0)
	c:/go/src/net/http/server.go:1265 +0x48
net/http.(*ServeMux).ServeHTTP(0xc083ea29c0, 0x110aa40, 0xc083e880a0, 0xc08383a0d0)
	c:/go/src/net/http/server.go:1541 +0x184
main.func·002(0x110aa40, 0xc083e880a0, 0xc08383a0d0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:228 +0xd3
net/http.HandlerFunc.ServeHTTP(0xc083ea4a00, 0x110aa40, 0xc083e880a0, 0xc08383a0d0)
	c:/go/src/net/http/server.go:1265 +0x48
main.func·005(0x110aa40, 0xc083e880a0, 0xc08383a0d0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:283 +0x1e4
net/http.HandlerFunc.ServeHTTP(0xc0823baa70, 0x110aa40, 0xc083e880a0, 0xc08383a0d0)
	c:/go/src/net/http/server.go:1265 +0x48
net/http.(*ServeMux).ServeHTTP(0xc083ea2a80, 0x110aa40, 0xc083e880a0, 0xc08383a0d0)
	c:/go/src/net/http/server.go:1541 +0x184
main.func·009(0x110aa40, 0xc083e880a0, 0xc08383a0d0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui_csrf.go:52 +0x419
net/http.HandlerFunc.ServeHTTP(0xc083ea4c00, 0x110aa40, 0xc083e880a0, 0xc08383a0d0)
	c:/go/src/net/http/server.go:1265 +0x48
main.func·006(0x110aa40, 0xc083e880a0, 0xc08383a0d0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:290 +0xb5
net/http.HandlerFunc.ServeHTTP(0xc0823baae0, 0x110aa40, 0xc083e880a0, 0xc08383a0d0)
	c:/go/src/net/http/server.go:1265 +0x48
net/http.serverHandler.ServeHTTP(0xc0820f9380, 0x110aa40, 0xc083e880a0, 0xc08383a0d0)
	c:/go/src/net/http/server.go:1703 +0x1a1
net/http.(*conn).serve(0xc084302000)
	c:/go/src/net/http/server.go:1204 +0xb5e
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 50 [IO wait]:
net.(*pollDesc).Wait(0xc0852e2170, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08229c008, 0xc0852e2060, 0xa5a7b0, 0x7, 0xba7b90, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc0852e2000, 0xc085fb2000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc083576010, 0xc085fb2000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
bufio.(*Reader).Read(0xc0820b4cc0, 0xc085fb2000, 0x1000, 0x1000, 0xffffffff, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:164 +0x141
main.(*WrappedConnection).Read(0xc0853200c0, 0xc085fb2000, 0x1000, 0x1000, 0x44f059, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/tls.go:123 +0x7a
net/http.(*liveSwitchReader).Read(0xc0852e40e8, 0xc085fb2000, 0x1000, 0x1000, 0xc08207e380, 0x0, 0x0)
	c:/go/src/net/http/server.go:214 +0xb2
io.(*LimitedReader).Read(0xc0853200e0, 0xc085fb2000, 0x1000, 0x1000, 0xc0895398e8, 0x0, 0x0)
	c:/go/src/io/io.go:408 +0xd5
bufio.(*Reader).fill(0xc0820f8060)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).ReadSlice(0xc0820f8060, 0xa, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:295 +0x25e
bufio.(*Reader).ReadLine(0xc0820f8060, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:324 +0x69
net/textproto.(*Reader).readLineSlice(0xc0863421b0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:55 +0xa5
net/textproto.(*Reader).ReadLine(0xc0863421b0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:36 +0x56
net/http.ReadRequest(0xc0820f8060, 0xc083eb6340, 0x0, 0x0)
	c:/go/src/net/http/request.go:598 +0xd2
net/http.(*conn).readRequest(0xc0852e40a0, 0x0, 0x0, 0x0)
	c:/go/src/net/http/server.go:586 +0x276
net/http.(*conn).serve(0xc0852e40a0)
	c:/go/src/net/http/server.go:1162 +0x6a5
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 34 [semacquire]:
sync.(*WaitGroup).Wait(0xc085320000)
	c:/go/src/sync/waitgroup.go:132 +0x170
github.com/syncthing/syncthing/internal/model.(*Model).ScanFolders(0xc082f39680, 0xc083d041e0)
	/go/src/github.com/syncthing/syncthing/internal/model/model.go:1197 +0x6fa
main.(*apiSvc).postDBScan(0xc083048480, 0x110aa40, 0xc0852e4000, 0xc085322000)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:785 +0x387
main.*apiSvc.(main.postDBScan)·fm(0x110aa40, 0xc0852e4000, 0xc085322000)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:158 +0x4c
net/http.HandlerFunc.ServeHTTP(0xc0823ba990, 0x110aa40, 0xc0852e4000, 0xc085322000)
	c:/go/src/net/http/server.go:1265 +0x48
net/http.(*ServeMux).ServeHTTP(0xc083ea2a20, 0x110aa40, 0xc0852e4000, 0xc085322000)
	c:/go/src/net/http/server.go:1541 +0x184
main.func·002(0x110aa40, 0xc0852e4000, 0xc085322000)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:230 +0x14f
net/http.HandlerFunc.ServeHTTP(0xc083ea4a00, 0x110aa40, 0xc0852e4000, 0xc085322000)
	c:/go/src/net/http/server.go:1265 +0x48
main.func·005(0x110aa40, 0xc0852e4000, 0xc085322000)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:283 +0x1e4
net/http.HandlerFunc.ServeHTTP(0xc0823baa70, 0x110aa40, 0xc0852e4000, 0xc085322000)
	c:/go/src/net/http/server.go:1265 +0x48
net/http.(*ServeMux).ServeHTTP(0xc083ea2a80, 0x110aa40, 0xc0852e4000, 0xc085322000)
	c:/go/src/net/http/server.go:1541 +0x184
main.func·009(0x110aa40, 0xc0852e4000, 0xc085322000)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui_csrf.go:63 +0x4ef
net/http.HandlerFunc.ServeHTTP(0xc083ea4c00, 0x110aa40, 0xc0852e4000, 0xc085322000)
	c:/go/src/net/http/server.go:1265 +0x48
main.func·006(0x110aa40, 0xc0852e4000, 0xc085322000)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:290 +0xb5
net/http.HandlerFunc.ServeHTTP(0xc0823baae0, 0x110aa40, 0xc0852e4000, 0xc085322000)
	c:/go/src/net/http/server.go:1265 +0x48
net/http.serverHandler.ServeHTTP(0xc0820f9380, 0x110aa40, 0xc0852e4000, 0xc085322000)
	c:/go/src/net/http/server.go:1703 +0x1a1
net/http.(*conn).serve(0xc085346000)
	c:/go/src/net/http/server.go:1204 +0xb5e
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 67 [IO wait]:
net.(*pollDesc).Wait(0xc0847d02f0, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08229c008, 0xc0847d01e0, 0xa5a7b0, 0x7, 0xba7b90, 0xc08388c0d3, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc0847d0180, 0xc084707000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc086248020, 0xc084707000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
bufio.(*Reader).Read(0xc0839362a0, 0xc084707000, 0x1000, 0x1000, 0xffffffff, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:164 +0x141
main.(*WrappedConnection).Read(0xc085320360, 0xc084707000, 0x1000, 0x1000, 0x44f059, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/tls.go:123 +0x7a
net/http.(*liveSwitchReader).Read(0xc08440e548, 0xc084707000, 0x1000, 0x1000, 0xc08207e000, 0x0, 0x0)
	c:/go/src/net/http/server.go:214 +0xb2
io.(*LimitedReader).Read(0xc085320380, 0xc084707000, 0x1000, 0x1000, 0xc0865898e8, 0x0, 0x0)
	c:/go/src/io/io.go:408 +0xd5
bufio.(*Reader).fill(0xc083936300)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).ReadSlice(0xc083936300, 0xa, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:295 +0x25e
bufio.(*Reader).ReadLine(0xc083936300, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:324 +0x69
net/textproto.(*Reader).readLineSlice(0xc086342360, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:55 +0xa5
net/textproto.(*Reader).ReadLine(0xc086342360, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:36 +0x56
net/http.ReadRequest(0xc083936300, 0xc0846584e0, 0x0, 0x0)
	c:/go/src/net/http/request.go:598 +0xd2
net/http.(*conn).readRequest(0xc08440e500, 0x0, 0x0, 0x0)
	c:/go/src/net/http/server.go:586 +0x276
net/http.(*conn).serve(0xc08440e500)
	c:/go/src/net/http/server.go:1162 +0x6a5
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 35 [IO wait]:
net.(*pollDesc).Wait(0xc0847d0470, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08229c008, 0xc0847d0360, 0xa5a7b0, 0x7, 0xba7b90, 0xc0846d20d3, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc0847d0300, 0xc0846d9000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc085f88008, 0xc0846d9000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
bufio.(*Reader).Read(0xc0835080c0, 0xc0846d9000, 0x1000, 0x1000, 0xffffffff, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:164 +0x141
main.(*WrappedConnection).Read(0xc0833ca0e0, 0xc0846d9000, 0x1000, 0x1000, 0x44f059, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/tls.go:123 +0x7a
net/http.(*liveSwitchReader).Read(0xc0821b4228, 0xc0846d9000, 0x1000, 0x1000, 0xc08207e000, 0x0, 0x0)
	c:/go/src/net/http/server.go:214 +0xb2
io.(*LimitedReader).Read(0xc0833ca100, 0xc0846d9000, 0x1000, 0x1000, 0xc0865dd8e8, 0x0, 0x0)
	c:/go/src/io/io.go:408 +0xd5
bufio.(*Reader).fill(0xc083508120)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).ReadSlice(0xc083508120, 0xa, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:295 +0x25e
bufio.(*Reader).ReadLine(0xc083508120, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:324 +0x69
net/textproto.(*Reader).readLineSlice(0xc0863424b0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:55 +0xa5
net/textproto.(*Reader).ReadLine(0xc0863424b0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:36 +0x56
net/http.ReadRequest(0xc083508120, 0xc08453c5b0, 0x0, 0x0)
	c:/go/src/net/http/request.go:598 +0xd2
net/http.(*conn).readRequest(0xc0821b41e0, 0x0, 0x0, 0x0)
	c:/go/src/net/http/server.go:586 +0x276
net/http.(*conn).serve(0xc0821b41e0)
	c:/go/src/net/http/server.go:1162 +0x6a5
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 68 [IO wait]:
net.(*pollDesc).Wait(0xc083a26170, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08229c008, 0xc083a26060, 0xa5a7b0, 0x7, 0xba7b90, 0xc08416a0d4, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc083a26000, 0xc08388d000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc085f88010, 0xc08388d000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
bufio.(*Reader).Read(0xc083508180, 0xc08388d000, 0x1000, 0x1000, 0xffffffff, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:164 +0x141
main.(*WrappedConnection).Read(0xc0853204c0, 0xc08388d000, 0x1000, 0x1000, 0x44f059, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/tls.go:123 +0x7a
net/http.(*liveSwitchReader).Read(0xc08440eb88, 0xc08388d000, 0x1000, 0x1000, 0xc08207e000, 0x0, 0x0)
	c:/go/src/net/http/server.go:214 +0xb2
io.(*LimitedReader).Read(0xc0853204e0, 0xc08388d000, 0x1000, 0x1000, 0xc08658d8e8, 0x0, 0x0)
	c:/go/src/io/io.go:408 +0xd5
bufio.(*Reader).fill(0xc083936360)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).ReadSlice(0xc083936360, 0xa, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:295 +0x25e
bufio.(*Reader).ReadLine(0xc083936360, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:324 +0x69
net/textproto.(*Reader).readLineSlice(0xc086342720, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:55 +0xa5
net/textproto.(*Reader).ReadLine(0xc086342720, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:36 +0x56
net/http.ReadRequest(0xc083936360, 0xc0846585b0, 0x0, 0x0)
	c:/go/src/net/http/request.go:598 +0xd2
net/http.(*conn).readRequest(0xc08440eb40, 0x0, 0x0, 0x0)
	c:/go/src/net/http/server.go:586 +0x276
net/http.(*conn).serve(0xc08440eb40)
	c:/go/src/net/http/server.go:1162 +0x6a5
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 36 [runnable]:
github.com/syncthing/syncthing/internal/model.func·008()
	/go/src/github.com/syncthing/syncthing/internal/model/model.go:1192 +0x23e
created by github.com/syncthing/syncthing/internal/model.(*Model).ScanFolders
	/go/src/github.com/syncthing/syncthing/internal/model/model.go:1195 +0x6b0

goroutine 28 [select]:
github.com/syncthing/syncthing/internal/ignore.(*Matcher).clean(0xc089df3a40, 0x68c61714000)
	/go/src/github.com/syncthing/syncthing/internal/ignore/ignore.go:161 +0x1d0
created by github.com/syncthing/syncthing/internal/ignore.New
	/go/src/github.com/syncthing/syncthing/internal/ignore/ignore.go:53 +0x177
```

--- a/cmd/syncthing/main.go
+++ b/cmd/syncthing/main.go
@@ -596,10 +596,6 @@ func syncthingMain() {
 		m.StartDeadlockDetector(20 * 60 * time.Second)
 	}
 
-	// GUI
-
-	setupGUI(mainSvc, cfg, m)
-
 	// Clear out old indexes for other devices. Otherwise we'll start up and
 	// start needing a bunch of files which are nowhere to be found. This
 	// needs to be changed when we correctly do persistent indexes.
@@ -611,7 +607,20 @@ func syncthingMain() {
 			}
 			m.Index(device, folderCfg.ID, nil, 0, nil)
 		}
+		// Routine to pull blocks from other devices to synchronize the local
+		// folder. Does not run when we are in read only (publish only) mode.
+		if folderCfg.ReadOnly {
+			l.Okf("Ready to synchronize %s (read only; no external updates accepted)", folderCfg.ID)
+			m.StartFolderRO(folderCfg.ID)
+		} else {
+			l.Okf("Ready to synchronize %s (read-write)", folderCfg.ID)
+			m.StartFolderRW(folderCfg.ID)
+		}
 	}
+
+	// GUI
+
+	setupGUI(mainSvc, cfg, m)
 
 	// The default port we announce, possibly modified by setupUPnP next.
 
@@ -635,18 +644,6 @@ func syncthingMain() {
 
 	connectionSvc := newConnectionSvc(cfg, myID, m, tlsCfg)
 	mainSvc.Add(connectionSvc)
-
-	for _, folder := range cfg.Folders() {
-		// Routine to pull blocks from other devices to synchronize the local
-		// folder. Does not run when we are in read only (publish only) mode.
-		if folder.ReadOnly {
-			l.Okf("Ready to synchronize %s (read only; no external updates accepted)", folder.ID)
-			m.StartFolderRO(folder.ID)
-		} else {
-			l.Okf("Ready to synchronize %s (read-write)", folder.ID)
-			m.StartFolderRW(folder.ID)
-		}
-	}
 
 	if cpuProfile {
 		f, err := os.Create(fmt.Sprintf("cpu-%d.pprof", os.Getpid()))

